config/locales/en.yml

```diff
@@ -635,3 +635,5 @@ Please follow the instructions there to make your enterprise visible on the Open
   price_graph: "Price graph"
   included_tax: "Included tax"
   remove_tax: "Remove tax"
+  og_description: "We begin from the ground up. With farmers and growers ready to tell their stories proudly and truly. With distributors ready to connect people with products fairly and honestly. With buyers who believe that better weekly shopping decisions can…"
```

Comment:
I would suggest to re-use the existing [i18n keys](https://github.com/ludivinecp/openfoodnetwork/blob/ogp-squashed/config/locales/en.yml#L139) telling the brand story. You could create a helper in `application_helper` (called `og_default_description`, for example), which puts the different strings from i18n together. In this way, we only need to maintain a single translation. I suppose combining headline and story would be enough here as a default.